integration/src/integration/java/pl/allegro/tech/hermes/integration/management/TopicManagementTest.java

```diff
@@ -103,4 +108,40 @@ public void shouldNotAllowOnCreatingSameTopicTwice() {
         // then
         assertThat(response).hasStatus(Response.Status.BAD_REQUEST).hasErrorCode(ErrorCode.TOPIC_ALREADY_EXISTS);
     }
+
+    @Test
+    public void shouldReturnTopicsThatAreCurrentlyTracked() {
+        // given
+        TopicName topic = createTrackedTopic("tracked", "topic");
+
+        // when
+        List<String> tracked = management.topic().list("", true);
+
+        // then
+        assertThat(tracked).containsOnly(topic.qualifiedName());
+    }
+
+    @Test
+    public void shouldReturnTopicsThatAreCurrentlyTrackedForGivenGroup() {
+        // given
+        TopicName topic1 = createTrackedTopic("tracked1", "topic1");
+        TopicName topic2 = createTrackedTopic("tracked2", "topic2");
+
+        // when
+        List<String> tracked = management.topic().list(topic1.getGroupName(), true);
+
+        // then
+        assertThat(tracked).contains(topic1.qualifiedName())
+                           .doesNotContain(topic2.qualifiedName());
+    }
+
+    private TopicName createTrackedTopic(String group, String name) {
+        TopicName topic = new TopicName(group, name);
+        operations.createGroup(topic.getGroupName());
+        operations.createTopic(topic.getGroupName(), topic.getName());
```

Comment:
this looks hacky, can we just create new `operations#createTopic(Topic)` method instead of creating and then updating?